tech-committee-charter.md

```diff
@@ -61,6 +61,30 @@ The development process will include a process for the TC to oversee and approve
 
 The TC and entire technical community will follow any processes as may be specified by the Cloud Native Computing Foundation relating to the intake and license compliance review of contributions, including the CNCF IP Policy.
 
+### Sponsorship Requirements
+
+The TC will establish and maintain sponsorship across the SIGs and subprojects of OpenTelemetry.
+
+SIGs will be classified into three classifications of Sponsorship requirement:
+
+* *Escalation Sponsorship*: The TC sponsor will be a primary means of escalation of cross-project concerns from the SIG. This communication is primarily
+  offline, e.g. via CNCF slack or issue communication. The TC sponsor will check-in with SIG leadership quarterly.  Additionally, the TC will aide the SIG
+  in developing quarterly road maps during this check-in.
+* *Guiding Sponsorship*: The TC sponsor is an active participant of the SIG. The sponsor will help direct the SIGs efforts to match the technical goals of OpenTelemetry.
+* *Leading Sponsorship*: The TC sponsor is an active member of SIG leadership, driving the direct success of the project or proposal.
```

Comment:
I understand from the name what this is, but the description is essentially the same as the guiding. I'd expand this to be explicit about the TC member being responsible for the SIG's completion of their charter goals. I'd also make it a requirement to at least attend SIG meetings, and run them in the absence of another maintainer.

Reply:
I just thought of one more point related to this. Is every SIG expected to have a TC leading sponsor, or is a spec sponsor enough to cover this role? I think any SIG that has a non-TC spec sponsor leader should at least have an escalation TC sponsor.

Reply:
Updated - Every SIG is required to at least have "escalation" sponsorship.

I like the language your'e using around Leading = owning outcomes.  I'll take a crack at updating the description shortly.

Reply:
Ok, fully updated, PTAL